### DISCUSSION TITLE
How to enable __float128 on IntelliSense?

### DISCUSSION THREAD

**Discussion by Phantom-Trigger** [microsoft/vscode-cpptools#9232] [Q&A] [answered]
My IntelliSense doesn't work with `__float128`. My platform is Windows, compiler is gcc11.2.0 from [winlibs](https://winlibs.com), C/C++ Extension Version is 1.10.8. I've tried all possible options for "Intelli Sense Mode" but all still have the same issue as #6574 (i.e. gives an IntelliSense error: identifier “__float128” is undefined). **But it do compiled and run successfully.** Did I misconfigured something?
![屏幕截图 2022-04-24 192616](https://user-images.githubusercontent.com/52826887/177983550-d721ebe7-1148-4e10-9efc-3944944c241d.png)

**Comment by sean-mcmanus** [comment, marked as answer]
This was moved to https://github.com/microsoft/vscode-cpptools/issues/9558
